### ISSUE TITLE
TEST_FILE

### ISSUE THREAD

**Issue by zcf131016** [BPYap/BERTSel#3]
Hello, we need a text_tsv and answer_pool when inference, which is different from the format of the training set and the validation set. Can you provide these two files for inference? Many thanks.

**Comment by BPYap** [comment]
Hi, you can find the test files for TrecQA and WikiQA in this [link](https://entuedu-my.sharepoint.com/:f:/g/personal/boonpeng001_e_ntu_edu_sg/EtuC6NQoy21PpuVN6Egfe6cBmUV8udJ6cu5qwsKymz5k-w?e=Xp3kNh). Hope it helps.

**Comment by zcf131016** [comment]
Thank you for your help, your test set and source code are very helpful to me. I may make further improvements based on this source code in the future, I hope you don't mind. At the same time, I want to ask whether you converted the test set of this style yourself or where you downloaded it, because I may test the model on other data sets later.

**Comment by BPYap** [comment]
Thanks for your interest in improving the source codes, any improvement is welcome :) 

Yes, I generate the test files myself as I thought separating the questions from the answer pool might be easier to work with. Unfortunately I have lost the script that does the formatting, but it should be relatively straightforward to implement.
